File: dayfive/afterbreak/SingleNumber.java
public class SingleNumber{
    public static void main (String []args){

    int[] number = {2,2,1};

    System.out.print(isSingle(number));

}
    public static int isSingle(int[] number){
    int value = number[0];
    int count = 0;
    
    for(int index = 0; index < number.length; index++){
        if (number[index] == number[value]){
            count++;   
    }
     
   }

    return count;

}




}

File: dayfive/afterbreak/SingleNumber.js
let number = [2,2,1];

function isSingle( number){
     value = number[0];
    count = 0;
    
    for( index = 0; index < number.length; index++){
        if (number[index] == number[value]){
            count++;   
    }
     
   }

    return count;

}

console.log (isSingle(number));[]
